Fixes bug found by Nikos Chantziaras
https://bugs.gentoo.org/show_bug.cgi?id=387949
Picked from upstream development branch.

>  main.c: In function 'main':
>  main.c:384:44: error: expected expression before ';' token
>  main.c:385:29: error: expected expression before ';' token

commit 84320a33d8f9027c70a8d972b69b1ffc824c5a8d
Author: Andrew Borodin <aborodin@vmail.ru>
Date:   Thu Oct 20 15:42:15 2011 +0400

    Ticket #????: cleanup.
    
    Fixed build with --disable-nls option.
    
    Signed-off-by: Andrew Borodin <aborodin@vmail.ru>

diff --git a/src/main.c b/src/main.c
index 9d442f2..e854719 100644
--- a/src/main.c
+++ b/src/main.c
@@ -381,8 +381,10 @@ main (int argc, char *argv[])
 
     /* We had LC_CTYPE before, LC_ALL includs LC_TYPE as well */
     (void) setlocale (LC_ALL, "");
+#ifdef ENABLE_NLS
     (void) bindtextdomain ("mc", LOCALEDIR);
     (void) textdomain ("mc");
+#endif
 
     if (!events_init (&error))
     {
